helm/charts/determined/templates/genai/genai-deployment.yaml

```
@@ -0,0 +1,75 @@
+{{- if .Values.genai.version }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: genai-deployment-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: genai-{{ .Release.Name }}
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: genai-{{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: genai-{{ .Release.Name }}
+        determined-system: master
+    spec:
+      priorityClassName: determined-system-priority
+      serviceAccount: genai-{{ .Release.Name }}
+      containers:
+      - name: genai-{{ .Release.Name }}
+        {{ $tag := (required "A valid .Values.genai.version entry required!" .Values.genai.version) }}
+        {{- /* genai.version is used for CI to override the appVersion. */ -}}
+        image: {{ .Values.imageRegistry }}/genai:{{ $tag }}
+        imagePullPolicy: "Always"
+        env:
+          - name: DET_MASTER
+            value: determined-master-service-{{ .Release.Name }}:{{ .Values.masterPort }}
+          - name: DB_NAME
+            value: lore
+          - name: DB_USER
+            value: postgres
+          - name: DB_PASSWORD
+            value: postgres
+          - name: DB_PORT
+            value: "5432"
```

these all can be changed by the user: https://github.com/determined-ai/determined/blob/main/helm/charts/determined/templates/master-config.yaml#L47

so,
```suggestion
          - name: DB_USER
            value: {{ .Values.db.user | quote }}
          - name: DB_PASSWORD
            value: {{ .Values.db.password | quote }}
          - name: DB_PORT
            value: {{ .Values.db.port }}
```

also note there's also SSL config which is not configured here: https://github.com/determined-ai/determined/blob/main/helm/charts/determined/templates/master-config.yaml#L57

I don't know how popular that SSL config is, so I won't insist on supporting it, but I'd suggest ticketing it or at least being aware of it as it's something that may surface for a sophisticated user.